aws/src/test/java/org/apache/iceberg/aws/s3/signer/TestS3SignRequestParser.java

```diff
@@ -126,7 +126,7 @@ public void roundTripSerde() {
                     "Content-Type",
                     Arrays.asList("application/json"),
                     "User-Agent",
-                    Arrays.asList("aws-sdk-java/2.20.18", "Linux/5.4.0-126")))
+                    Arrays.asList("aws-sdk-java/2.20.131", "Linux/5.4.0-126")))
```

Comment:
we don't really need to update those. It doesn't matter what headers the test is using

Reply:
Thanks. Reverted the change